### PR TITLE
[Docu] website: Documentation enhanced

### PR DESCRIPTION
**Pull request by Pavan-SAP**
Misc improvements to some documentation.
Mainly operator metrics part and some anchors.

### CODE REVIEW

File: website/content/en/docs/usage/operator-metrics.md [hunk 1]
```diff
@@ -7,56 +7,72 @@ description: >
...
+The CAP Operator includes built-in [Prometheus metrics](https://pkg.go.dev/github.com/prometheus/client_golang/prometheus) that enable users to effectively monitor and analyze the operator's performance. These metrics can provide insights into resource usage, potential issues, and overall operator health. The metrics are accessible at the `/metrics` endpoint on port `9090` of both the [Controller](docs/concepts/operator-components/controller/) and the [Subscription Server](docs/concepts/operator-components/subscription-server/).
 
-{{% pageinfo %}}
+## Controller Metrics
+
+The Controller emits several types of metrics, including:
+
+- **Standard Go Metrics**: These metrics are provided by the [Prometheus Go client](https://pkg.go.dev/github.com/prometheus/client_golang/prometheus/collectors#WithGoCollectorRuntimeMetrics) and include runtime statistics.
+- **Workqueue Metrics**: These metrics are relevant to the resources being reconciled and are based on the [MetricsProvider](https://pkg.go.dev/k8s.io/client-go/util/workqueue#MetricsProvider).
+- the following specific metrics:
```
Comment (anirudhprasad-sap) [inline website/content/en/docs/usage/operator-metrics.md:20]:
```suggestion
- [Specific metrics](<Link to the below section>)
```

File: website/content/en/docs/usage/operator-metrics.md [hunk 2]
```diff
@@ -7,56 +7,72 @@ description: >
...
+
+### Specific Metrics
+
+1. **Reconcile Errors**
 ```
 cap_op_reconcile_errors{kind="CAPApplication",name="my-app",namespace="app"} 11
```
Comment (anirudhprasad-sap) [inline website/content/en/docs/usage/operator-metrics.md:26]:
```suggestion
cap_op_reconcile_errors{kind="CAPApplication",name="my-app",namespace="app"}
```
Not sure if it is a typo or not. I can see a number in all metrics expressions

Comment (Pavan-SAP) [reply website/content/en/docs/usage/operator-metrics.md:26]: The whole expression is s sample including labels and numbers

**Review by anirudhprasad-sap — APPROVED** [review]
👍

**Pull request merged**